src/main/java/org/alliancegenome/curation_api/services/validation/dto/fms/GeneExpressionAnnotationFmsDTOValidator.java

```diff
@@ -0,0 +1,179 @@
+			response.addErrorMessage("geneId - ", ValidationConstants.REQUIRED_MESSAGE + " (" + geneExpressionFmsDTO.getGeneId() + ")");
+		} else {
+			Gene expressionAnnotationSubject = geneService.findByIdentifierString(geneExpressionFmsDTO.getGeneId());
+			if (expressionAnnotationSubject == null) {
+				response.addErrorMessage("geneId - ", ValidationConstants.INVALID_MESSAGE + " (" + geneExpressionFmsDTO.getGeneId() + ")");
+			} else {
+				geneExpressionAnnotation.setExpressionAnnotationSubject(expressionAnnotationSubject);
+			}
+		}
+
+		if (ObjectUtils.isEmpty(geneExpressionFmsDTO.getDateAssigned())) {
+			response.addErrorMessage("dateAssigned - ", ValidationConstants.REQUIRED_MESSAGE + " (" + geneExpressionFmsDTO.getDateAssigned() + ")");
+		} else {
+			OffsetDateTime creationDate = null;
+			try {
+				creationDate = OffsetDateTime.parse(geneExpressionFmsDTO.getDateAssigned());
+			} catch (DateTimeParseException e) {
+				response.addErrorMessage("dateAssigned", ValidationConstants.INVALID_MESSAGE + " (" + geneExpressionFmsDTO.getDateAssigned() + ")");
+			}
+			geneExpressionAnnotation.setDateCreated(creationDate);
+		}
+
+		if (ObjectUtils.isEmpty(geneExpressionFmsDTO.getAssay())) {
+			response.addErrorMessage("assay - ",  ValidationConstants.REQUIRED_MESSAGE + " (" + geneExpressionFmsDTO.getAssay() + ")");
+		} else {
+			MMOTerm expressionAssayUsed = mmoTermService.findByCurie(geneExpressionFmsDTO.getAssay());
+			if (expressionAssayUsed == null) {
+				response.addErrorMessage("assay - ", ValidationConstants.INVALID_MESSAGE + " (" + geneExpressionFmsDTO.getAssay() + ")");
+			} else {
+				geneExpressionAnnotation.setExpressionAssayUsed(expressionAssayUsed);
+			}
+		}
+
+		if (ObjectUtils.isEmpty(geneExpressionFmsDTO.getWhereExpressed())) {
+			response.addErrorMessage("whereExpressed - ",  ValidationConstants.REQUIRED_MESSAGE + " (" + geneExpressionFmsDTO.getWhereExpressed() + ")");
+		} else {
+			String whereExpressedStatement = geneExpressionFmsDTO.getWhereExpressed().getWhereExpressedStatement();
+			if (ObjectUtils.isEmpty(whereExpressedStatement)) {
+				response.addErrorMessage("whereExpressed - whereExpressedStatement",  ValidationConstants.REQUIRED_MESSAGE + " (" + geneExpressionFmsDTO.getWhereExpressed().getWhereExpressedStatement() + ")");
+			} else {
+				geneExpressionAnnotation.setWhereExpressedStatement(whereExpressedStatement);
+			}
+		}
+
+		if (ObjectUtils.isEmpty(geneExpressionFmsDTO.getWhenExpressed())) {
+			response.addErrorMessage("whenExpressed - ",  ValidationConstants.REQUIRED_MESSAGE + " (" + geneExpressionFmsDTO.getWhenExpressed() + ")");
+		} else {
+			String stageName = geneExpressionFmsDTO.getWhenExpressed().getStageName();
+			if (ObjectUtils.isEmpty(stageName)) {
+				response.addErrorMessage("whenExpressed - whenExpressedStageName",  ValidationConstants.REQUIRED_MESSAGE + " (" + geneExpressionFmsDTO.getWhenExpressed().getStageName() + ")");
```

Comment:
🚫 **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.whitespace.SingleSpaceSeparatorCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Use a single space to separate non-whitespace characters.